src/Common/src/Interop/Unix/System.Native/Interop.EnumerateInterfaceAddresses.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public unsafe struct LinkLayerAddressInfo
+        {
+            public int InterfaceIndex;
+            public fixed byte AddressBytes[8];
+            public byte NumAddressBytes;
+            private byte __pading;
```

Comment:
No, this is 8-byte aligned: the `ushort` on line 17 bumps the size up to 16 bytes.